resources/js/components/Members.vue

```diff
@@ -229,35 +229,23 @@
                     return arr.indexOf(elem) == pos;
                 }).join(", ");
             },
-            csvlist: function () {
-                var targetList = this.members;
-                if (this.filterList) {
-                    targetList = this.members.filter(e => e.filtered);
-                }
-                const rows = targetList.map(r => {
-                    return {
-                        "surname": r.user.surname,
-                        "given name": r.user.givenname,
-                        "email": r.user.email,
-                        "role": r.role.label,
-                        "notes": r.notes,
-                        "start date": r.start_date,
-                        "end date": r.end_date,
-                        "office": r.user.office?r.user.office.replace(/ \$ /g, "\n"):null,
-                        "title": r.user.title
-                    }
-                });
-                return rows;
-                // let row_str = 'Surname, GivenName, Label, Notes, Start Date, End Date\n'
-                // row_str += rows.join('\n');
-
-                // console.log(row_str);
-
-                // const link = document.createElement("a");
-                // const file = new Blob([row_str], {type: 'text/csv'});
-                // link.href = URL.createObjectURL(file);
-                // link.download = '' + this.group.group_title + '.csv';
-                // link.click();
+            csvlist() {
+                return this.filteredList.map((member) => ({
```

Comment:
the filter flag is actually a response to a user interaction -users can hit "filter list" and then select a subset of items. So, we don't want to break that functionality. If we want to include the previous members, the approach would be something more like this (obviously the variable names suck):
```
let targetList = this.filteredList;
if(this.filterList) {
	targetList = this.filteredList.filter(e=>e.filtered);
}
const rows = targetList.map ... snip
```

Reply:
I think I'm following: you're saying there's a `filtered` prop on the individual group member and a `filterList` boolean which indicates whether to use this filtered prop. But, the `filteredList` computed prop is distinct from these. 

Did I get that right?